codegen.yml

```
@@ -0,0 +1,25 @@
+overwrite: true
+schema:
+  - 'https://demo.vendure.io/shop-api'
```

👍 now we are using this vendure endpoint

```suggestion
  - 'https://readonlydemo.vendure.io/shop-api'
```

---

👍

---

I have pushed a possible solution. Let me know if it's ok 👍